index.html

```diff
@@ -4,10 +4,37 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">   
     <link rel="stylesheet" href="style.css">
     <title>Portfolio</title>
 </head>
 <body>
-    <div class="header"></div>
+    <div class="header">
+        <img src="/image/images.png" width="100px">
+        <a href="#" class="fa fa-bars"></a>
```

Comment:
- [ ] Kindly wrap the humburger button in a `nav` tag. The `nav` tag for semantic reasons.